frontend/src/static/js/main.js

```diff
@@ -254,15 +254,12 @@ window.app = new window.Vue({
       return 'http://reposense.org';
     },
 
-    getRepoSenseVersionLink() {
+    getRepoSenseUserGuideVersionLink() {
```

Comment:
I think it might be sufficient to use a shorter name - `getUserGuideLink` ?
From the code itself, it is relatively clear that the version is being used.

Reply:
I think it is better to be a bit explicit with the function name. Maybe, I can make it - `getUserGuideVersionLink`?

Reply:
>Maybe, I can make it - getUserGuideVersionLink?

I would personally still learn towards `getUserGuideLink`, but this is also ok.

Reply:
@fzdy1914 your views on this?

Reply:
`getUserGuideVersionLink` is better, since we may implement a direct link to the user guide in the future.